src/main/java/com/sprint/mission/discodeit/entity/UserStatus.java

```diff
@@ -0,0 +1,25 @@
+package com.sprint.mission.discodeit.entity;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.time.Instant;
+import java.util.UUID;
+
+@AllArgsConstructor
+@Getter
+@Setter
+public class UserStatus {
+    private UUID id;
+    private UUID userid;
+    private Instant lastLogin;
+
+    public boolean isLogin(){
+        // getEpochSecond 흐른 초를 반환 (5분 x 60초 = 300)
+        if (lastLogin.getEpochSecond() < 300) {
+            return true;
+        }
+        return false;
```

Comment:
@yun0ro 로직이 잘못 되었어요. 현재 시간과의 차이를 계산해야 하는데 위의 로직은 단순히 흐른 초를 반환하고 있어요.

```suggestion
    public boolean isLogin(){
        Instant now = Instant.now();
        long diffSeconds = now.getEpochSecond() - lastLogin.getEpochSecond();
        return diffSeconds < 300; // 5분(300초) 이내
```

Reply:
오류 발견 감사합니다...! ^^